packages/apm/src/hubextensions.ts

```diff
@@ -33,16 +33,17 @@ function traceHeaders(): { [key: string]: string } {
  * and attach a `SpanRecorder`. If it's of type `SpanContext` and there is already a `Span` on the Scope,
  * the created Span will have a reference to it and become it's child. Otherwise it'll crete a new `Span`.
  *
- * @param span Already constructed span which should be started or properties with which the span should be created
+ * @param spanOrSpanContext Already constructed span which should be started or properties with which the span should be created
```

Comment:
> Already constructed span which should be started

Isn't the span started (`start_timestamp` set) in the constructor? Currently it is impossible to have a span that is not started, unless one manually unsets `start_timestamp` after object instantiation.